ui/app/scripts/task/views/definitions.html

```diff
@@ -21,21 +21,12 @@
   <tbody>
     <tr dir-paginate="item in pageable.items | itemsPerPage: pageable.pageSize | filter:filterQuery" total-items="pageable.total" ng-show="!item.inactive">
       <td>{{item.name}}</td>
-      <td xd-format-stream='item.definition'></td>
+      <td xd-format-stream='item.dslText'></td>
       <td>{{item.status}}</td>
```

Comment:
Not part of this PR, but is it normal to see the status of the task as `deploying` always though the task app says it started? cc @mminella @cppwfs